sonic_installer/bootloader/aboot.py

```diff
@@ -163,7 +163,10 @@ def get_binary_image_version(self, image_path):
             return None
         return IMAGE_PREFIX + version.strip()
 
-    def verify_binary_image(self, image_path):
+    def verify_image_platform(self, image_path):
+        return os.path.isfile(image_path)
```

Comment:
The `verify_image_platform` function will not be doing as much on Arista platforms than it is on ONIE but this is something we can add at a later time.
I think for now it's fine to just check if the file is there.